lib/data/entity.js

```diff
@@ -102,7 +131,7 @@ const parseUpdateJson = (entity, properties, body) => {
     if (k === 'label') label = v;
     else if (propSet.has(k))
       data[k] = isBlank(v) ? '' : v;
-    else
+    else // TODO: maybe this should be a different problem.
```

Comment:
In the submission entity code, I use this `entityViolation` error a bunch, which is a 409 conflict error. Some of the problems there, and here with JSON parsing, would probably be better as the new 400.28 error about `invalidEntity`, but I haven't gone and tried to change those yet.